dipy/tracking/tests/test_streamline.py

```diff
@@ -516,6 +517,46 @@ def test_unlist_relist_streamlines():
         assert_array_equal(streamlines[i], streamlines2[i])
 
 
+def test_deform_streamlines():
+    # streamlines needs to be a list of streamlines
+    A = streamlines = np.array([[1, 2, 3], [1, 2, 3.]])
+    streamlines = [A for i in range(10)]
+    # Create Random deformation field
+    deformationField = np.empty((4, 4, 4, 3))
+    shape = deformationField.shape
+    for i in range(shape[0]):
+        for j in range(shape[1]):
+            for k in range(shape[2]):
+                deformationField[i][j][k] = np.random.rand(3)
```

Comment:
Would you mind using `randn` instead? That way we get deformations in both negative in positive directions.

Reply:
done